php/php.editor/src/org/netbeans/modules/php/editor/verification/ReturnTypeHintError.java

```diff
@@ -133,17 +140,17 @@ private void checkReturnType(Map<ASTNode, Set<ReturnStatement>> returnStatements
         "ReturnTypeHintErrorVoidDesc=\"{0}\" cannot return anything"
     })
     private void checkVoidAndNeverReturnStatements(Set<ReturnStatement> statements, String name, List<Hint> hints) {
-        if (Type.VOID.equals(name) || isNeverType(name)) {
+        if (isVoidType(name) || isNeverType(name)) {
             // check empty return statement
-            statements.forEach((statement) -> {
+            for (ReturnStatement statement: statements) {
```

Comment:
Why this change?

Reply:
I think I see now - because of the `CancelSupport`, right?

Reply:
Yes, you are right.